caesars-palace/blocks/slider-card/slider-card.css

```diff
@@ -110,8 +128,12 @@
     background: #fff;
     position: relative;
     padding: 16px;
-    font-size: 16px;
-    height: 190px;
+    font-size: 12px;
```

Comment:
I think there is a variable for that size. Please check and use it if it's the case